nncf/nncf_network.py

```diff
@@ -672,18 +684,22 @@ def get_insertion_point_graph(self) -> InsertionPointGraph:
             ip_graph_node_type = ip_graph_node[InsertionPointGraph.NODE_TYPE_NODE_ATTR]
             if ip_graph_node_type == InsertionPointGraphNodeType.OPERATOR:
                 nncf_graph_node_ref = ip_graph_node[InsertionPointGraph.REGULAR_NODE_REF_NODE_ATTR]
-                op_exec_context = nncf_graph_node_ref[NNCFGraph.OP_EXEC_CONTEXT_NODE_ATTR]
-                op_name = op_exec_context.operator_name
-                scope = op_exec_context.scope_in_model
-                op_arch = OPERATOR_METATYPES.get_operator_metatype_by_op_name(op_name)
-                module = self.get_module_by_scope(scope)
-                if module is not None:
-                    subtype = op_arch.determine_subtype(containing_module=module)
-                    if subtype is not None:
-                        op_arch = subtype
+                op_arch = self.get_op_arch_by_graph_node(nncf_graph_node_ref)
                 ip_graph_node[InsertionPointGraph.OPERATOR_METATYPE_NODE_ATTR] = op_arch
         return ip_graph
 
+    def get_op_arch_by_graph_node(self, nncf_graph_node_ref):
+        op_exec_context = nncf_graph_node_ref[NNCFGraph.OP_EXEC_CONTEXT_NODE_ATTR]
+        op_name = op_exec_context.operator_name
+        scope = op_exec_context.scope_in_model
+        op_arch = OPERATOR_METATYPES.get_operator_metatype_by_op_name(op_name)
```

Comment:
Will OperatorSubtype resolution work here? For instance, it is impossible to tell DW-conv and conv apart based on the function name only, you need the module for that.

Reply:
subtype resolution happens after that:
https://github.com/openvinotoolkit/nncf/pull/518/files/8244eb755114545fe58c6bef20cb69f5298ea80d#diff-acec2cd12abd71c59568f29765b23819248f804127cf70a32640c80b9ab733dbR696-R701

Reply:
Talk about selective vision XD